### DISCUSSION TITLE
DDT4All Dacia duster 2016 odometer

### DISCUSSION THREAD

**Discussion by Oran67** [cedricp/ddt4all#1082] [Q&A]
Hi !
I turned off the Tpms function. After that i got problem with (se picture). I cant se the diesel consumption and the meter for consumtion doesent move. It always presents full tank. How can i fix this ? 
![IMG_2023](https://github.com/cedricp/ddt4all/assets/166980177/227a5c11-d5c6-4383-b69c-f544370aa839)

**Comment by shrlnm** [comment]
Hi @Oran67 
What did you do exactly? Which commands did you use? Which ECU?

**Reply by Oran67** [reply]
DDt4all
Hi, i used X79 Duster
cluster /Tdb
cluster _x52_X67_X79_X92_v5.0

When i scan for fault codes now......
Maybe i had them initialy, and nothing to do about ?

-----------------------------------------------------------
-----------------------------------------------------------
Car Scanner ELM OBD2

DTC report

Selected brand: Dacia


============1==============

C1608

Raw code: 5608

ECU: Power steering (11 bit)

Status: Pending fault present, Test has not been completed, Validated fault present at time of request


============2==============

B1413(55)

Raw code: 941355

ECU: Dashboard/Instrument cluster (11 bit)

Status: Test failed, Test failed (current drive cycle), Confirmed, Test failed since last DTC clear, Warning indicator requested


============3==============

B1413(55)

Raw code: 941355

ECU: Dashboard/Instrument cluster (11 bit)

Status: Test failed, Test failed (current drive cycle), Confirmed, Test failed since last DTC clear, Warning indicator requested

Odometer: 161002 km

============4==============

C1A60(7B)

Raw code: 5A607B

ECU: BCM (11 bit)

Status: Confirmed, Test not completed since last DTC clear, Test failed since last DTC clear, Warning indicator requested


============5==============

C1A82(68)

Raw code: 5A8268

ECU: BCM (11 bit)

Status: Confirmed, Test not completed since last DTC clear, Test failed since last DTC clear, Warning indicator requested


============6==============

C1A83(68)

Raw code: 5A8368

ECU: BCM (11 bit)

Status: Confirmed, Test not completed since last DTC clear, Test failed since last DTC clear, Warning indicator requested


============7==============

C1A84(68)

Raw code: 5A8468

ECU: BCM (11 bit)

Status: Confirmed, Test not completed since last DTC clear, Test failed since last DTC clear, Warning indicator requested


============8==============

C1A85(68)

Raw code: 5A8568

ECU: BCM (11 bit)

Status: Confirmed, Test not completed since last DTC clear, Test failed since last DTC clear, Warning indicator requested


============9==============

C1A72(7A)

Raw code: 5A727A

ECU: BCM (11 bit)

Status: Confirmed, Test not completed since last DTC clear, Test failed since last DTC clear, Warning indicator requested


============10==============

C1A73(7A)

Raw code: 5A737A

ECU: BCM (11 bit)

Status: Confirmed, Test not completed since last DTC clear, Test failed since last DTC clear, Warning indicator requested


============11==============

C1A74(7A)

Raw code: 5A747A

ECU: BCM (11 bit)

Status: Confirmed, Test not completed since last DTC clear, Test failed since last DTC clear, Warning indicator requested


============12==============

C1A75(7A)

Raw code: 5A757A

ECU: BCM (11 bit)

Status: Confirmed, Test not completed since last DTC clear, Test failed since last DTC clear, Warning indicator requested


============13==============

B1266(81)

Raw code: 926681

ECU: BCM (11 bit)

Status: Confirmed, Test failed since last DTC clear, Warning indicator requested


============14==============

C1A60(7B)

Raw code: 5A607B

ECU: BCM (11 bit)

Status: Confirmed, Test not completed since last DTC clear, Test failed since last DTC clear, Warning indicator requested

Odometer: 154456 km

============15==============

C1A82(68)

Raw code: 5A8268

ECU: BCM (11 bit)

Status: Confirmed, Test not completed since last DTC clear, Test failed since last DTC clear, Warning indicator requested

Odometer: 161000 km

============16==============

C1A83(68)

Raw code: 5A8368

ECU: BCM (11 bit)

Status: Confirmed, Test not completed since last DTC clear, Test failed since last DTC clear, Warning indicator requested

Odometer: 161000 km

============17==============

C1A84(68)

Raw code: 5A8468

ECU: BCM (11 bit)

Status: Confirmed, Test not completed since last DTC clear, Test failed since last DTC clear, Warning indicator requested

Odometer: 161000 km

============18==============

C1A85(68)

Raw code: 5A8568

ECU: BCM (11 bit)

Status: Confirmed, Test not completed since last DTC clear, Test failed since last DTC clear, Warning indicator requested

Odometer: 161000 km

**Comment by Oran67** [comment]
![IMG_2012](https://github.com/cedricp/ddt4all/assets/166980177/8b5eae69-5ed1-4536-a0ae-ac2143fdd30c)

**Reply by KarelSvo** [reply]
give my the ddt4all/logs/elm_ddt  and ecu_ddt files
The bottom 7 control units in your photo do not belong to the Duster Phase 2. Have you experimented with them too?
Read errors only with PyRen or PyCLIP. Then no ghost errors will be displayed.

**Comment by Oran67** [comment]
[ecu_ddt.txt](https://github.com/cedricp/ddt4all/files/15047098/ecu_ddt.txt)
[elm_ddt.txt](https://github.com/cedricp/ddt4all/files/15047099/elm_ddt.txt)

**Reply by KarelSvo** [reply]
I can't find any cluster;write command > 243;2E and no BCM;write command > 745;2E in your ecu_ddt file
No changes were confirmed by these two ECUs.

**Reply by Oran67** [reply]
- Hi... im a beginner so i dont understand these commands. How/were do i write them 😲

**Comment by KarelSvo** [comment]
give screens.txt file

**Reply by Oran67** [reply]
Ahh ok ill try these commands ✔️

**Reply by KarelSvo** [reply]
You write that you have changed the TPMS function in the cluster.
But according to your log files, no change commands were sent.

**Reply by Oran67** [reply]
I reinstalled because the program hangs. Found a newer version.

**Comment by Oran67** [comment]
[dusterphase1.txt](https://github.com/cedricp/ddt4all/files/15049328/dusterphase1.txt)
